src/flamegraph/mod.rs

```diff
@@ -450,8 +454,15 @@ where
         merge::frames(lines, false)?
     } else {
         // Sort lines by default.
-        let mut lines: Vec<&str> = lines.into_iter().collect();
-        lines.sort_unstable();
+        let mut lines: Vec<&str> = if let Some(base) = &opt.base {
+            lines
+                .into_iter()
+                .filter_map(|line| line.rfind(base).map(|cut| &line[cut..]))
+                .collect()
+        } else {
+            lines.into_iter().collect()
+        };
+        lines.sort();
```

Comment:
Why change this from `sort_unstable` to `sort`?

Reply:
A few lines above in the `reverse` handling code there is a `sort` so when I was editing this I'm sure I wasn't looking carefully at the diff and I was going off what I could see in my editor.

Which sort do you prefer?

Reply:
`sort_unstable` is generally faster and better for memory-use, so definitely prefer that unless you _specifically_ need a stable sort.